manifests/deploy/openshift/olm/metering.catalogsourceconfig.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: operators.coreos.com/v1
+kind: CatalogSourceConfig
+metadata:
+  name: installed-redhat-metering-operators-openshift-metering
+  namespace: openshift-marketplace
+spec:
+  targetNamespace: openshift-metering
+  packages: metering-ocp
+
```

Comment:
We reverted this commit, so this needs to be rebased against the current master

Reply:
Thank you for calling MCI... please hold for rebase.